### PR TITLE
Updated "Exercise 4".

### PR DESCRIPTION
**Pull request by NavneetDeshmukh**
Updated "Exercise 4".

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/design-patterns/ex4gsioverload/Step1.en.md [hunk 1]
```diff
@@ -18,25 +18,25 @@ KeySchema=[{AttributeName=GSI_1_PK,KeyType=HASH},{AttributeName=GSI_1_SK,KeyType
...
+Let's take a closer look at the create-table command. You are creating a table named employees. The partition key on the table is *PK* and it holds the **employee ID**. The sort key is *SK*, which contains a derived value that you choose in the Python script. (We'll revisit this shortly.) You will create a global secondary index on this table and name it *GSI_1*; This will be an overloaded global secondary index. The partition key on the global secondary index is *GSI_1_PK*, and holds the same value as the sort key *SK* on the base table. The *GSI_1* sort key value is **name**, and its attribute name is *GSI_1_SK*.
```
Comment (switch180) [inline content/design-patterns/ex4gsioverload/Step1.en.md:25]: ** should be replaced with a tick mark `, but you need to check the source docx to be sure

File: content/design-patterns/ex4gsioverload/_index.en.md [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
+The following screenshot shows the design of the employees table. The attribute called *PK* has the employee ID, which is prefixed by the letter e. The hash sign (#) is used as a separator between the entity type identifier (e) and the actual employee ID. The *SK* is  an overloaded attribute, and has either current title, previous title, or the keyword *master*, which denotes the primary item for the employee that holds most of their important attributes. The *GSI_1_PK* attribute includes the title or the name of the employee. The re-use of a given global secondary index for multiple entity types such as employees, employee locations, and employee titles lets us simplify our management of the DynamoDB table because we only need to monitor and pay for one global secondary index as opposed to three separate indexes.
+
+TODO :  Please add a one-point border around this image.
```
Comment (switch180) [inline content/design-patterns/ex4gsioverload/_index.en.md:21]: you. can remove this todo

File: content/design-patterns/ex4gsioverload/_index.en.md [hunk 3]
```diff
@@ -1,22 +1,23 @@
...
+The following screenshot shows the design of the employees table. The attribute called *PK* has the employee ID, which is prefixed by the letter e. The hash sign (#) is used as a separator between the entity type identifier (e) and the actual employee ID. The *SK* is  an overloaded attribute, and has either current title, previous title, or the keyword *master*, which denotes the primary item for the employee that holds most of their important attributes. The *GSI_1_PK* attribute includes the title or the name of the employee. The re-use of a given global secondary index for multiple entity types such as employees, employee locations, and employee titles lets us simplify our management of the DynamoDB table because we only need to monitor and pay for one global secondary index as opposed to three separate indexes.
```
Comment (switch180) [inline content/design-patterns/ex4gsioverload/_index.en.md:19]: numerous ticks missing

File: content/design-patterns/ex4gsioverload/Step3.en.md [hunk 4]
```diff
@@ -72,21 +72,21 @@ You will get the result:
...
...ordinator,Information Technology Director,Information Technology Manager,IT Support Manager,IT Support Specialist,IT Systems Administrator,Java Developer,Junior Software Engineer,Management Information Systems Director,.NET Developer,Network Architect,Network Engineer,Network Systems Administrator,Programmer,Programmer Analyst,Security Specialist,Senior Applications Engineer,Senior Database Administrator,Senior Network Architect,Senior Network Engineer,Senior Network System Administrator,Senior Programmer,Senior Programmer Analyst,Senior Security Specialist,Senior Software Engineer,Senior Support Specialist,Senior System Administrator,Senior System Analyst,Senior System Architect,Senior System Designer,Senior Systems Analyst,Senior Systems Software Engineer,Senior Web Administrator,Senior Web Developer,Software Architect,Software Developer,Software Engineer,Software Quality Assurance Analyst,Support Specialist,Systems Administrator,Systems Analyst,System Architect,Systems Designer,Systems Software Engineer,Technical Operations Officer,Technical Support Engineer,Technical Support Specialist,Technical Specialist,Telecommunications Specialist,Web Administrator,Web Developer,Webmaster*
```
Comment (switch180) [inline content/design-patterns/ex4gsioverload/Step3.en.md:79]: remove the single asterisks around the job titles to remove italics. See docx for formatting reference.

File: content/design-patterns/ex4gsioverload/Step1.en.md (hunk 1 above)
Comment (NavneetDeshmukh) [reply content/design-patterns/ex4gsioverload/Step1.en.md]: Done

File: content/design-patterns/ex4gsioverload/_index.en.md (hunk 3 above)
Comment (NavneetDeshmukh) [reply content/design-patterns/ex4gsioverload/_index.en.md]: fixed

File: content/design-patterns/ex4gsioverload/Step3.en.md (hunk 4 above)
Comment (NavneetDeshmukh) [reply content/design-patterns/ex4gsioverload/Step3.en.md]: done

**Comment by NavneetDeshmukh** [comment]
Fixed syntax

**Comment by switch180** [comment]
LGTM. Merging.

**Pull request merged**